euler_support/urdf/euler_macro.xacro

```diff
@@ -1,182 +1,175 @@
+        </geometry>
+      </collision>
+    </link>
+    <link name="${prefix}rear_sick_s300">
+      <visual>
+        <geometry>
+          <mesh filename="package://euler_support/meshes/visual/sick_s300_mounting_assy.stl"/>
+        </geometry>
+        <material name="">
+          <color rgba="0.75 0.75 0 1"/>
+        </material>
+      </visual>
+      <collision>
+        <geometry>
+          <mesh filename="package://euler_support/meshes/collision/sick_s300_mounting_assy.stl"/>
+        </geometry>
+      </collision>
+    </link>
+    <joint name="${prefix}frame_left" type="fixed">
+      <parent link="${prefix}base_link"/>
+      <child link="${prefix}frame_left"/>
+      <origin xyz="0 0 0" rpy="0 0 0"/>
+      <axis xyz="0 0 0"/>
+    </joint>
+    <joint name="${prefix}frame_right" type="fixed">
+      <parent link="${prefix}base_link"/>
+      <child link="${prefix}frame_right"/>
+      <origin xyz="0 0 0" rpy="0 0 0"/>
+      <axis xyz="0 0 0"/>
+    </joint>
+    <joint name="${prefix}frame_top" type="fixed">
+      <parent link="${prefix}base_link"/>
+      <child link="${prefix}frame_top"/>
+      <origin xyz="0 0 0" rpy="0 0 0"/>
+      <axis xyz="0 0 0"/>
+    </joint>
+    <joint name="${prefix}to_vetex" type="fixed">
+      <parent link="${prefix}base_link"/>
+      <child link="${prefix}vetex_base_link"/>
+      <origin xyz="0 0 0" rpy="0 0 0"/>
+      <axis xyz="0 0 0"/>
+    </joint>
+    <joint name="${prefix}sick_nav350" type="fixed">
+      <parent link="${prefix}base_link"/>
+      <child link="${prefix}sick_nav350"/>
+      <origin xyz="-0.239 -0.648 2.551" rpy="0 0 0"/>
+      <axis xyz="0 0 0"/>
+    </joint>
+    <joint name="${prefix}base_to_localization" type="fixed">
+      <parent link="${prefix}base_link"/>
+      <child link="${prefix}localization_frame"/>
+      <origin xyz="-0.239 -0.648 0" rpy="0 0 0"/>
+      <axis xyz="0 0 0"/>
+    </joint>
+    <joint name="${prefix}front_sick_s300" type="fixed">
+      <parent link="${prefix}base_link"/>
+      <child link="${prefix}front_sick_s300"/>
+      <origin xyz="0.193 -0.617 0.532" rpy="0 0 -0.75047"/>
+      <axis xyz="0 0 0"/>
+    </joint>
+    <joint name="${prefix}rear_sick_s300" type="fixed">
+      <parent link="${prefix}base_link"/>
+      <child link="${prefix}rear_sick_s300"/>
+      <origin xyz="-2.280 0.617 0.532" rpy="0 0  -3.96178"/>
+      <axis xyz="0 0 0"/>
+    </joint>
+    <!--euler to motoman link-->
+    <joint name="${prefix}platform_to_arm" type="fixed">
+      <parent link="${prefix}base_link"/>
+      <child link="${prefix}motoman_base_link"/>
+      <origin xyz="0.067518 0 2.016" rpy="3.1415 0 0"/>
+      <axis xyz="0 0 0"/>
+    </joint>
+    <!--motoman to end effector link-->
+    <joint name="${prefix}motoman_link_t_to_eef_base" type="fixed">
+      <parent link="${prefix}motoman_link_t"/>
+      <child link="${prefix}eef_eef_base"/>
```

Comment:
Out of curiosity, what did you use to format the xml?

Reply:
xmllint --format `old_file` > `new_file`